docs/src/content/docs/getting-started.mdx

```diff
@@ -85,3 +85,34 @@ If you forget anything important, Starlight will let you know.
 Once you have created and customized your Starlight website, you can deploy it to a web server or hosting platform of your choice including Netlify, Vercel, GitHub Pages and many more.
 
 [Learn about deploying an Astro site in the Astro docs.](https://docs.astro.build/en/guides/deploy/)
+
+## Updating Starlight
+
+Starlight is an Astro integration, and is updated like any `@astrojs/*` integration:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+# upgrade Starlight with npm
+npm upgrade @astrojs/starlight
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+# upgrade Starlight with pnpm
+pnpm upgrade @astrojs/starlight
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+# upgrade Starlight with yarn
+yarn upgrade @astrojs/starlight
+```
```

Comment:
Updating the commands here as per the discussion we just had for the 2.6 blog post.

````suggestion
```sh
# upgrade Starlight with npm
npm install @astrojs/starlight@latest
```

</TabItem>
<TabItem label="pnpm">

```sh
# upgrade Starlight with pnpm
pnpm upgrade @astrojs/starlight --latest
```

</TabItem>
<TabItem label="Yarn">

```sh
# upgrade Starlight with yarn
yarn upgrade @astrojs/starlight --latest
```
````